helm/loki/values.yaml

```diff
@@ -233,67 +252,17 @@ loki:
         memory: 3Gi
         cpu: 500m
 
-  loki:
-    image:
-      repository: giantswarm/loki
-    memcached:
-      chunk_cache:
-        enabled: false
-        host: "memcached-app.loki.svc"
-        service: "memcache"
-        batch_size: 256
-        parallelism: 10
-      results_cache:
-        enabled: false
-        host: "memcached-app.loki.svc"
-        service: "memcache"
-        timeout: "500ms"
-        default_validity: "24h"
-    limits_config:
```

Comment:
Removed the limits_config in favor or values set in shared_configs because defautls should be fine